IdentityCore/src/requests/sdk/msal/MSIDDefaultBrokerResponseHandler.m

```diff
@@ -83,6 +87,17 @@ - (MSIDBrokerResponse *)brokerResponseFromEncryptedQueryParams:(NSDictionary *)e
         return nil;
     }
     
+    // In case MSAL requests AT POP but Broker response Bearer, MSAL returns error to user
+    if ([authScheme isMemberOfClass:MSIDAuthenticationSchemePop.class])
```

Comment:
For my understanding, any reason to use ```isMemberOfClass``` vs ```isMemberOfClass```

Reply:
@kaisong1990: Sorry I didn't understand your question, can you add more detail?

Reply:
maybe Kai meant isMemberOfClass vs isKindOfClass?

Reply:
My bad, I copied it twice, updated the question

Reply:
Hi Kai, for the context, we have MSIDAuthenticationScheme (default - Bearer protocol) and then we have MSIDAuthenticationSchemePop (new protocol) inherits from the default. We're trying to verify if the auth scheme get from resumeState of MSAL matching the auth scheme from broker's response.
_isMemberOfClass_ check if the object is an instance of the class
_isKindOfClass_ check if the object is an instance of the class or any class inheriting from it.
Currently, there isn't any class inheriting from Pop, so I guess either of the methods should work. But in the future, if a new protocol inherits from AuthPop, the check _isKindOfClass_ will not work. So _isMemberOfClass_ is a better choice.

Reply:
Thanks for the clarification